### ISSUE TITLE
Nelson please help me <3

### ISSUE THREAD

**Issue by Game4Freak** [SmartlyDressedGames/Unturned-3.x-Community#1618]
I really didn't want to write this here but its my only option currently.
Last week I got a game ban while helping someone fixing an issue with a plugin of mine.
I already wrote a appeal to Battleye but they don't answer :(
I can 100% prove that I didn't cheat. I have absolutely no reason to cheat.
This game ban prevents me from helping others with plugins and it prevents me from updating and uploading workshop content that I make and need for my plugins.
And it prevents me from having fun with others playing :(

It would be very nice if you could help me get in contact with Battleye to solve this issue <3

**Comment by Trojaner** [comment]
Game4Freak has made one of the biggest and most creative plugins for this game. I really don't think he would cheat and strongly support that his case should be revised.

**Comment by ConorJM51** [comment]
Game4Freak is very supportive about Unturned, and loves making unique plugins to help with demand from the Roleplay community since we have nothing to support us with Role play, he has no reason to cheat. I do agree with Trojaner, this case should be revised.

**Comment by Johnanater** [comment]
Agreeing with the others here, I've dealt with false bans from bad game companies and know how frustrating it can be. Hopefully they get back to you eventually :(

You might be able to ask for a GDPR Subject Access Request to get information on your ban, or at least get them to look into it. That's what I did with mine to get it competently looked into and overturned.

**Comment by joeymisfit** [comment]
This reminds me of back in the old days when there was that huge Battleye ban spree, when it was just a mistake, I was one of the first that had gotten banned and nobody believed me until everyone else started getting randomly banned. @SDGNelson If you could possibly review @Game4Freak's  Battleye ban as I believe there has been a mistake, he's done nothing but good for the Unturned Community with his plugin creations. Thank you!

**Comment by archie426** [comment]
BE are useless with appeals. Now that HWID banning has been introduced I think there should be a better appealing system

**Comment by SDGNelson** [comment]
I do find this really surprising considering your position in the community. Can you link your Steam profile please? I will personally check into what happened.

**Comment by Game4Freak** [comment]
Yes thats my profile and thanks <3

**Comment by SDGNelson** [comment]
Okay. I will let you all know when I hear back on the ticket.

**Comment by CatzMR** [comment]
Players in China also have a large number of bans. Many players who have not played the game for a long time are banned after opening the game again. The reason is unknown. I hope to check the recent bans and check whether there is a problem of wrong closure. by: I use Google Translate because I do n’t understand English

**Comment by surv0013** [comment]
> Players in China also have a large number of bans.

Unban only for famous people, the Chinese will not be heard. :(

**Comment by CatzMR** [comment]
But I suspect that Battleye may have mistakenly identified some players recently

**Comment by Johnanater** [comment]
It's possible that there's a false positive with the recent update

**Comment by CatzMR** [comment]
@Johnanater  I think so too.

**Comment by tankirules** [comment]
My friend got game banned right after a big false positive ban wave, sent multiple emails to BE, no response ever. We've lost hope in battleeye, especially because he's an active workshop contributer and can no longer publish his mods

**Comment by SDGNelson** [comment]
Today we heard back on the ticket after the BattlEye team have investigated the situation. It looks like the bans on you and twelve other players have been lifted. This is definitely concerning and I am sorry for the false positive! I will follow up if there are any new details.

**Comment by CatzMR** [comment]
@SDGNelson  Many players have been mistakenly blocked by the BattlEye. We recommend checking the recent ban.

**Comment by SDGNelson** [comment]
Considering BattlEye reviewed the case I think we can trust that other recent bans are correct. They clearly examined other bans when finding the other twelve false positives. BattlEye support also handles appeals to double-check the bans and would have handled this.

Players complaining about being banned has scaled with the number of active players lately (more players stuck at home), so I do not think there has been a mistaken ban wave.

**Comment by CatzMR** [comment]
@SDGNelson They said that they have not played this game for a long time, but they were banned after returning to play. Is this a chase?

**Comment by SDGNelson** [comment]
They probably had cheat software still installed from when they last played the game.

**Comment by CatzMR** [comment]
https://github.com/CatzMR/Unturned-BUG
@SDGNelson  Please download Unturned intelligent Chinese patch to see if it will be banned, you need to put two .exe into the game root directory

**Comment by CatzMR** [comment]
I hope I can check if BattlEye has identified these two .exe files that are put into the root directory of the game as hackers

**Comment by SDGNelson** [comment]
It is unusual to replace the game executable. BattlEye would kick for using a modified exe, but only ban if they have been flagged by BattlEye staff as a cheat. What features do the localization patch need that are unsupported by the base game?

**Comment by CatzMR** [comment]
I hope you can check whether BattlEye has marked these two .exes as cheating software. In China, there are players who have reacted and are banned by BattlEye because of the Chinese version.

**Comment by SDGNelson** [comment]
Following my other comment:

> I will follow up with you here if there are any new details.

There has not been any other word from BattlEye about plugin concerns, so I think this is closed for the meantime. I will continue keeping an eye out for unusual bans. Usually I would say feel free to comment and I will re-open, but unless Game4Freak gets banned again (uh oh) it probably makes more sense in a new issue.
